Jenkinsfile

```diff
@@ -51,7 +51,23 @@ pipeline {
     stage('Unit Test') {
       steps {
         ansiColor('xterm') {
-          sh '''npm run test -- --maxWorkers=2'''
+          sh '''npm run test -- --maxWorkers=2 --coverage'''
```

Comment:
Could we move it to another PR so we make sure that this does not get too many resources, or that it really works independently?

Reply:
I openend this PR: https://github.com/dcos/dcos-ui/pull/2801